build/copyright/ccri-dstl-copyright-2017.txt

```diff
@@ -1,6 +0,0 @@
-Copyright (c) ${project.inceptionYear}-2016 ${owner}
-Portions Crown Copyright (c) 2017 Dstl
```

Comment:
i think the new files are right, it's just the diff view isn't very good... here's what we have after this PR:
`ccri` -> default, ccri, latest year
`ccri-dstl-copyright` -> ccri latest year, dstl 2017
`ccri-dstl-copyright-2016` -> ccri latest year, dstl 2016
`dstl-copyright-2016` -> dstl 2016
`dstl-copyright-2017` -> dstl 2017
`ibm` -> ibm 2017
`ibm-ccri` -> ibm 2017, ccri latest year